src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,15 @@
-export const TodoList = () => {};
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoInfo } from '../TodoInfo';
+
+interface ToDoListProps {
+  todos: Todo[];
+}
+
+export const TodoList: React.FC <ToDoListProps> = ({ todos }) => (
```

Comment:
Remove the unnecessary space before the angle bracket.
```suggestion
export const TodoList: React.FC<ToDoListProps> = ({ todos }) => (
```